modules/meeting/app/components/meetings/side_panel/attachments_component.html.erb

```diff
@@ -0,0 +1,34 @@
+<%=
+  component_wrapper do
+    render(Primer::OpenProject::SidePanel::Section.new) do |section|
+      section.with_title { t(:label_attachment_plural) }
+      section.with_description { I18n.t('meeting.attachments.text') }
+      section.with_counter { @meeting.attachments.count }
+
+      flex_layout do |attachments_container|
+        attachments_container.with_row(mt: 2) do
+          resource = ::API::V3::Meetings::MeetingRepresenter.new(@meeting, current_user: User.current, embed_links: false)
+          helpers.list_attachments(resource,
+                                   inputs: {
+                                     allowUploading: true,
+                                     showTimestamp: false,
+                                     externalUploadButton: "#meetings-add-attachments"
+                                   })
+        end
+
+        attachments_container.with_row(mt: 1) do
+          render(Primer::Beta::Button.new(
+            scheme: :link,
+            color: :default,
+            underline: false,
+            font_weight: :bold,
+            id: "meetings-add-attachments"
+          )) do |button|
+            button.with_leading_visual_icon(icon: "op-attachmentadd")
+            t("js.label_add_attachments")
+          end
+        end
```

Comment:
I guess this whole block should be the `with_footer_button` slot, right? Then you can get rid of most of the system arguments, propably also the `mt: 1`.